### DISCUSSION TITLE
crash

### DISCUSSION THREAD

**Discussion by bf69** [indomus/forum#272] [HUB: Home Assistant]
ciao, stamattina mi sono ritrovato HA bloccato, senza nessun motivo. il log è pieno di errori:
mi sembra di capire che il problema arrivi dal file "fullchain.pem".. cosa posso fare?
grazie
`2021-05-05 07:34:45 ERROR (MainThread) [homeassistant.config] Invalid config for [http]: file not readable for dictionary value @ data['http']['ssl_certificate']. Got '/home/homeassistant/dehydrated/certs/casafabio.duckdns.org/fullchain.pem'
file not readable for dictionary value @ data['http']['ssl_key']. Got '/home/homeassistant/dehydrated/certs/casafabio.duckdns.org/privkey.pem'. (See /home/homeassistant/.homeassistant/configuration.yaml, line 42). Please check the docs at https://www.home-assistant.io/integrations/http
2021-05-05 07:34:45 ERROR (MainThread) [homeassistant.setup] Setup failed for http: Invalid config.
2021-05-05 07:34:45 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of system_log. Setup failed for dependencies: http
2021-05-05 07:34:45 ERROR (MainThread) [homeassistant.setup] Setup failed for system_log: Could not set up all dependencies.
2021-05-05 07:34:45 WARNING (Recorder) [homeassistant.components.recorder.util] The system could not validate that the sqlite3 database at //mnt/usbhdd/home-assistant_v2.db was shutdown cleanly`

**Comment by snakuzzo** [comment]
Il motivo c'è..ed è indicato appunto negli errori.
Probabilmente sono stati "rigenerati" i certificati con i permessi di root e se è così. l'utente con cui avvii HA non può leggerli

**Reply by bf69** [reply]
si, stanotte penso che si siano rigenerati i certificati, ma come lo fa ogni mese... perchè stavolta è andato storto? cosa posso fare?

**Reply by snakuzzo** [reply]
gli cambi i permessi

**Reply by bf69** [reply]
selezionando i file e modificando le proprietà? e tra un mese?

**Reply by snakuzzo** [reply]
Anche.
Comunque verifica lo script che fa...cosa ha loggato...

**Reply by bf69** [reply]
ok, grazie, stasera ci provo..

**Reply by bf69** [reply]
è bastato digitare "sudo chmod -R 0777 /home/homeassistant" per aggiornare i permessi e tutto è ripartito...

**Reply by snakuzzo** [reply]
...ok...anche se dal mio punto di vista i permessi a 777 non vanno "MAI" messi :)

**Reply by bf69** [reply]
è meglio 775? perchè? una guida di indomus mi sembra che li impostava a 777..

**Reply by snakuzzo** [reply]
sinceramente non ricordo all'epoca come sono stati impostati, ma devi impostarli in modo tale che siano solo leggibili dall'utente che fa partire Home Assistant.
Se i file dei certificati sono di root, i permessi andrebbero impostati con `644`

**Reply by bf69** [reply]
ok grazie
